### DISCUSSION TITLE
Issues switching default editor

### DISCUSSION THREAD

**Discussion by ramiroaraujo** [johnlindquist/kit#1347] [Errors]
I use Webstorm, and have it configured in `.env` as
```
KIT_EDITOR=/Applications/WebStorm.app/Contents/MacOS/webstorm
```
I want to switch to the EAP version (`/Applications/WebStorm 2023.2 EAP.app/Contents/MacOS/webstorm`), but it just keeps opening the old one. Quoted, unquoted, doesn't mother. Tried restarting the app, force repair, the same. I even wrote invalid values/paths and Kit immediately fails due to the wrong path, so it's reading the changes.

Not a big issue, but an annoyance :)
Any help is appreciated

**Comment by johnlindquist** [comment]
My thoughts:

Follow the "Create a shell script" section here:

https://www.jetbrains.com/help/webstorm/working-with-the-ide-features-from-command-line.html#standalone

And try pointing the shell script to the EAP. Then point the .env to the shell script. Maybe that will work?

**Reply by ramiroaraujo** [reply]
Yeah, already done that, I usually open the IDE from the terminal with that script :/

**Reply by johnlindquist** [reply]
@ramiroaraujo If you add `console.log(env.KIT_EDITOR)` to the beginning of a script then check the logs?

You can also hack around with console.logs in `global.edit`:

`~/.kit/platform/darwin.js`

then check the `~/.kit/logs/kit.log` to see what's going on.

> Note: After you change a file in the SDK (like `darwin.js`), the change won't take affect until you open the prompt once then close it again.
